edx_solutions_api_integration/courses/views.py

```diff
@@ -2522,3 +2524,50 @@ def get(self, request, course_id, usage_key_string):  # pylint: disable=W0613
         }
 
         return Response(result, status=status.HTTP_200_OK)
+
+
+def _update_blocks(_block, _blocks):
+    children = []
+    for block_id in _block['children']:
+        _blocks[block_id]['id'] = block_id
+        children.append(_blocks[block_id])
+
+    _block['children'] = children
+    for b in _block['children']:
+        _update_blocks(b, _blocks)
+
+
+class CoursesTree(SecureAPIView):
+    """
+    **Use Case**
+
+        CoursesTree returns a course tree for a list of comma seperated course ids.
+
+
+    **Example Request**
+
+        GET /api/courses/tress?course_ids=CA/CS102/2018,CA/CS104/2019
```

Comment:
Can you add an example of how the response will look like? Since we'r changing the default structure